### DISCUSSION TITLE
admins.json wiping up

### DISCUSSION THREAD

**Discussion by Keylio00** [kus/cs2-modded-server#45] [Q&A]
I dunno why admins.json file wiping up after start of the server (win.bat). Can someone help me please?

**Comment by mavproductions** [comment]
admins.json is in custom_files? Perhaps using both CSS and Matchzy admin.jsons in tandem. Some redundancy, but might fix your problem.

**Comment by kus** [comment]
As @mavproductions pointed out, do you have your customisations in the `custom_files` folder? If not read about it [here](https://github.com/kus/cs2-modded-server?tab=readme-ov-file#custom-files).

When the server starts it merges all the files from this repo into your server to keep it updated. So if you make any changes in `/game/csgo/addons/counterstrikesharp/configs/admins.json` for example they will be overwritten. You need to copy any file you want to modify and recreate the directory structure in the `custom_files` folder. As once the server has merged the files from this repo it then merges the files from `custom_files` over the top.

You can look at `custom_files_example` which is the actual files I use to run my server and my admins are set at `/custom_files_example/addons/counterstrikesharp/configs/admins.json`.

Please let me know if you have any issues or this solves it.
